apps/juxtaposition-ui/src/models/logs.js

```diff
@@ -0,0 +1,35 @@
+const { Schema, model } = require('mongoose');
+
+const actionEnum = [
+	'REMOVE_POST',
+	'IGNORE_REPORT',
+	'LIMIT_POSTING',
+	'TEMP_BAN',
+	'PERMA_BAN',
+	'UNBAN',
+	'UPDATE_USER',
+	'MAKE_COMMUNITY',
+	'UPDATE_COMMUNITY',
+	'DELETE_COMMUNITY'
+];
+
+const auditLogSchema = new Schema({
+	actor: Number,
+	action: {
+		type: String,
+		enum: actionEnum
+	},
+	target: String,
```

Comment:
Something to consider for later is having seperate fields for the different ID types (PID, community id, post id, etc.) that currently share this field. I'm thinking once Postgres happens, it'll allow us to use foreign keys here.
I'm not sure if it's worth changing for this PR, but it's good to keep in mind that the frontend also assumes this shared field idea.

Reply:
Fair enough!